src/View/StringTemplate.php

```diff
@@ -323,4 +324,68 @@ protected function _formatAttribute($key, $value, $escape = true)
 
         return $key . '="' . ($escape ? h($value) : $value) . '"';
     }
+
+    /**
+     * Adds a class and returns a unique list either in array or space separated
+     *
+     * ### Options
+     *
+     * - `useIndex` if you are inputting an array with an 'element' other than 'class'.
+     *              Also setting to 'false' will manipulate the whole array (default is 'class')
+     * - `asString` Setting this to `true` will return a space separated string (default is `false`)
```

Comment:
I also think we should keep the API here simple, and rather manually `implode(' ', $array)` afterwards, easy enough, right?